server/namespace/classifer.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2017 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package namespace
+
+import "github.com/pingcap/pd/server/core"
+
+var (
```

Comment:
Declare them as constant values.

Reply:
Actually it is intentional. User may want to change default namespace name.

Reply:
em, I think it is no need to let the user change the default name.

Reply:
okay.